src/main/java/underdogs/devbie/auth/controller/interceptor/annotation/NoValidate.java

```diff
@@ -0,0 +1,11 @@
+package underdogs.devbie.auth.controller.interceptor.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface NoValidate {
+}
```

Comment:
```suggestion
@Target({ElementType.METHOD, ElementType.TYPE})
@Retention(RetentionPolicy.RUNTIME)
public @interface NoValidate {
}
```
클래스도 지원해주면 더 편의성 있는 Annotation이 될 것같아요!

Reply:
생각지도 못했는데 좋은 것 같습니다!